packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -435,7 +492,14 @@ class LongPressDraggable<T extends Object> extends Draggable<T> {
     Axis? axis,
     Widget? childWhenDragging,
     Offset feedbackOffset = Offset.zero,
+    @Deprecated(
+      'Use dragAnchorStrategy instead. '
+      'Replace "dragAnchor: DragAnchor.child" with "dragAnchorStrategy: childDragAnchorStrategy". '
+      'Replace "dragAnchor: DragAnchor.pointer" with "dragAnchorStrategy: pointerDragAnchorStrategy". '
+      'This feature was deprecated after v2.1.0-10.0.pre.'
+    )
     DragAnchor dragAnchor = DragAnchor.child,
+    DragAnchorStrategy? dragAnchorStrategy,
```

Comment:
We may need a test for this. Since Draggable already has test coverage, We can probably just create the LongPressDraggable with dragAnchorStrategy and make sure the same callback is passed into Draggable